roofit/roofitcore/inc/RooAbsCollection.h

```diff
@@ -263,12 +263,12 @@ class RooAbsCollection : public TObject, public RooPrintable {
     removeAll();
   }
 
-  inline Int_t getSize() const { 
-    // Return the number of elements in the collection
+  /// Return the number of elements in the collection
+  inline Int_t getSize() const {
     return _list.size();
   }
-  
-  inline RooAbsArg *first() const { 
+
+  inline RooAbsArg *first() const {
```

Comment:
```suggestion

  /// Return the first element in this collection.
  inline RooAbsArg *first() const {
```
Could you move the docstring while you are at it? (and put a period at the end of the sentence).